pkg/errors/errors.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+//nolint
```

Comment:
Again, why nolint?

Reply:
> Tast project

spellchecker thinks the project name should be `Taste` instead of the keyword `Tast` :upside_down_face: